docs/pages/development/getting-started.md

```diff
@@ -55,7 +55,9 @@ Once you have registered all of the iOS devices you would like to develop on, yo
 
 </Tabs>
 
-and [installing the resulting build on your device](/build/internal-distribution.md).
+After creating the first build, [install it on your device](/build/internal-distribution.md).
+
+> **Note**: If you are using a library that contains native APIs, for example, [`expo-secure-store`](/versions/latest/sdk/securestore/), you will have to rebuild the development client. This is because the native code of the library is not included in the development client automatically when installing the library as a dependency to your project.
```

Comment:
```suggestion
> **Note**: If you add a library that contains native APIs to your project, for example, [`expo-secure-store`](/versions/latest/sdk/securestore/), you will have to rebuild the development client. This is because the native code of the library is not included in the development client automatically when installing the library as a dependency to your project.
```